### DISCUSSION TITLE
ControlNet, move enable toggle to tab header

### DISCUSSION THREAD

**Discussion by quasiblob** [Mikubill/sd-webui-controlnet#662] [Ideas]
It is not so easy to see if ControlNet is enabled or disabled.

Could it be possible to move enable/disable toggle to the tab header?

Something like this:

![image](https://user-images.githubusercontent.com/13034046/227657069-7fadcdb8-3d65-4383-ba3c-539df2ee0173.png)

**Comment by PowerfulShell** [comment]
bump

**Comment by enternalsaga** [comment]
any news?

**Comment by PowerfulShell** [comment]
come on, this is such great improvement.... why no update
